iree_tests/configs/config_gpu_rocm_models.json

```
@@ -0,0 +1,17 @@
+{
+    "config_name": "rocm_llvm_task",
+    "iree_compile_flags" : [
+      "--iree-hal-target-backends=rocm",
+      "--iree-rocm-target-chip=gfx90a",
+      "--iree-opt-const-eval=false",
+      "--verify=false",
+      "--iree-codegen-transform-dialect-library=/mnt/share/amdsharks/sai/specs/attention_and_matmul_spec.mlir"
+    ],
+    "iree_run_module_flags": [
+      "--device=hip"
+    ],
+    "skip_compile_tests": [],
+    "skip_run_tests": [],
+    "expected_compile_failures": ["resnet50"],
```

nit: could put this on a new line to make future lines possible to add without unrelated diffs

```suggestion
    "expected_compile_failures": [
      "resnet50",
    ],
```

compare:
```diff
    "expected_compile_failures": [
+     "mobilenet",
      "resnet50",
    ],
```

with:
```diff
-   "expected_compile_failures": ["resnet50"],
+   "expected_compile_failures": [
+     "mobilenet",
+     "resnet50",
+   ],
```